cli/tests/e2e/snapshots/test_dependency_aware_rules/test_dependency_aware_rules/rulesdependency_awarego-sca.yaml-dependency_awarego_toolchain/results.txt

```diff
@@ -50,16 +50,17 @@ SEMGREP_USER_AGENT_APPEND="pytest" SEMGREP_SETTINGS_FILE="<MASKED>" SEMGREP_VERS
             "lockfile": "targets/dependency_aware/go_toolchain/go.mod"
           },
           "reachability_rule": true,
-          "reachable": false,
+          "reachable": true,
           "sca_finding_schema": 20220913
         },
-        "severity": "WARNING"
+        "severity": "WARNING",
+        "validation_state": "NO_VALIDATOR"
       },
-      "path": "targets/dependency_aware/go_toolchain/go.mod",
+      "path": "targets/dependency_aware/go_toolchain/sca.go",
```

Comment:
This seems unrelated. Maybe merge develop and you should not need this?

Reply:
Indeed. @mjambon fixed this test snapshot later in `Update test snapshot (how did it get in?)` #9119. 

I also asked about this in our [#team-supply-chain](https://semgrepinc.slack.com/archives/C032JB2HRDK/p16984501901239590) channel.